app/build.gradle

```diff
@@ -76,18 +76,25 @@ android {
     flavorDimensions "stage"
     productFlavors {
         production {
-            applicationId "com.nimbl3"
+            applicationId "co.nimblehq"
         }
 
         staging {
-            applicationId "com.nimbl3.staging"
+            applicationId "co.nimblehq.staging"
+        }
+
+        development {
+            applicationId "co.nimblehq.dev"
         }
     }
 
     sourceSets {
```

Comment:
Should there also be a sourceSet for `production`?

Reply:
@Tuubz nope, we don't need source set for custom build flavor https://github.com/nimblehq/android-templates/pull/48#discussion_r534038256